.github/ISSUE_TEMPLATE/other-en.md

```diff
@@ -0,0 +1,11 @@
+---
+name: Other
+about: Ask a question, leave a comment, seek guidance
+title: "[OTHER] Summary of the purpose"
+labels: ''
+assignees: ''
+
+---
+
+## Describe the purpose of this issue
```

Comment:
```suggestion
## Describe the purpose of this issue
<!-- Mandatory -->
```

Reply:
Having it in the preceding section header is sufficient.